soh/src/overlays/effects/ovl_Effect_Ss_Stick/z_eff_ss_stick.c

```diff
@@ -50,20 +52,21 @@ u32 EffectSsStick_Init(PlayState* play, u32 index, EffectSs* this, void* initPar
 
 void EffectSsStick_Draw(PlayState* play, u32 index, EffectSs* this) {
     GraphicsContext* gfxCtx = play->state.gfxCtx;
+    Player* player = GET_PLAYER(play);
     s32 pad;
 
     OPEN_DISPS(gfxCtx);
 
     Matrix_Translate(this->pos.x, this->pos.y, this->pos.z, MTXMODE_NEW);
-
-    if (!LINK_IS_ADULT) {
+    
+    if (Player_HoldsStick(player)) {
```

Comment:
_(I should preface this by saying I'm not 100% clear on how the Deku Stick code works or why you've changed this.)_

I have similar concerns here too. Especially since [an older commit](https://github.com/HarbourMasters/Shipwright/pull/3002/commits/b72c1682f50d2fae10fc9f78dc9060ff4f363e88#diff-c071dd0ce6a1a1b51bde4fbcfccd91871c40a08457c4e99c535a8cccf17b4c8dR65) did have you separating its function into whether or not Alternate Equipment Loading is activated, before you changed how checking for this worked.

Should this maybe be using the Timeless Equipment CVar in the check?
```suggestion
    if (!LINK_IS_ADULT || (CVarGetInteger("gTimelessEquipment", 0) && Player_HoldsStick(player))) {
```
If I'm not mistaken, this way it will use the vanilla condition if the CVar is off, and allow the alternative behavior as Adult Link only if the CVar is on and the alternative condition is met.

Reply:
(Oh, and, similarly in the `EffectSsStick_Init` function.)
```c
    Player* player = GET_PLAYER(play);
    if (CVarGetInteger("gTimelessEquipment", 0))
        ageInfoEntry = Player_HoldsStick(player) + drawInfo;
```